src/Features/Core/Portable/ExtractMethod/MethodExtractor.CodeGenerator.cs

```diff
@@ -41,8 +45,6 @@ protected abstract partial class CodeGenerator<TStatement, TExpression, TNodeUnd
 
             protected CodeGenerator(InsertionPoint insertionPoint, SelectionResult selectionResult, AnalyzerResult analyzerResult, TCodeGenerationOptions options, bool localFunction)
             {
-                Contract.ThrowIfFalse(insertionPoint.SemanticDocument == analyzerResult.SemanticDocument);
```

Comment:
much cleaner.  analzyer results don't have a document, so we dn't have to ensure these are in sync.